server.lua

```diff
@@ -46,33 +99,17 @@ local DISCORD_NAME = "DreamLife RP BOT"
 local DISCORD_IMAGE = "https://img.freepik.com/premium-vector/cute-robot-waving-hand-cartoon-illustration_138676-2744.jpg?w=2000"
 local LOG_FOOTER = '[DreamLife RP LOGS]'
 
----@param data { type: string, message: string, postalJobState?: table }
-RegisterServerEvent('dream-postal:server:log', function(data)
-    local src = source
+---@param data { type: string, message: string, postalJobState?: table, source: int }
```

Comment:
I don't believe we're passing postalJobState to this function anymore

```suggestion
---@param data { type: string, message: string, source: int }
```